src/hooks/swap/useSyncTokenDefaults.test.ts

```diff
@@ -0,0 +1,93 @@
+import { TradeType } from '@uniswap/sdk-core'
+import { SupportedChainId } from 'constants/chains'
+import { nativeOnChain } from 'constants/tokens'
+import { useAtomValue } from 'jotai/utils'
+import { Field, stateAtom, Swap, swapAtom } from 'state/swap'
+import { renderHook } from 'test'
+
+import { DAI, USDC_MAINNET } from '../../constants/tokens'
+import useSyncTokenDefaults, { TokenDefaults } from './useSyncTokenDefaults'
+
+const web3React = require('@web3-react/core')
```

Comment:
use an import:

    import * as web3React from '@web3-react/core'

The jest.mock should still work, and you can avoid overriding the lint rule.